bootstrap-salt.sh

```diff
@@ -1324,6 +1324,18 @@ __check_dpkg_architecture() {
         "amd64")
             error_msg=""
             ;;
+        "arm64")
+            if [ "$_CUSTOM_REPO_URL" != "null" ]; then
+                warn_msg="Support for arm64 is experimental, make sure the custom repository used has the expected structure and contents."
+            else
+                # Saltstack official repository does not yet have arm64 metadata,
+                # use amd64 repositories on arm64, since all pkgs are arch-independent
+                __REPO_ARCH="amd64"
+                __REPO_ARCH_DEB="[arch=$__REPO_ARCH]"
```

Comment:
I think we should declare some default value for `__REPO_ARCH_DEB` and redefine it here.
Let it be `__REPO_ARCH_DEB='deb'` before the case statement and here we would assign it
`deb [arch=$__REPO_ARCH]` value.